src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -1300,15 +1300,17 @@ class CosmasGoogleCloudServiceTest {
     }
 
     private fun deleteFile(fileId: String, fileName: String, time: Long,
-                           info: ProjectInfo = projectInfo()): StreamRecorder<DeleteFileResponse> {
-        val deleteFileRecorder: StreamRecorder<DeleteFileResponse> = StreamRecorder.create()
-        val deleteFileRequest = DeleteFileRequest.newBuilder()
-                .setInfo(info)
+                           info: ProjectInfo = projectInfo()): StreamRecorder<DeleteFilesResponse> {
```

Comment:
Pls add a couple of tests for batch delete.

Reply:
done